### DISCUSSION TITLE
Desktop - add account switching feature

### DISCUSSION THREAD

**Discussion by oldcityws** [community/community#59482] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Desktop should allow users / owners of organizations to switch between the accounts, and import repos / projects between the accounts.

**Comment by JennPlothow** [comment]


**Reply by oldcityws** [reply]
I do not understand why a "Resume Template" docx file is posted on this thread.

**Comment by C6C23** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Desktop should allow users / owners of organizations to switch between the accounts, and import repos / projects between the accounts.
